src/components/Scrollable/Scrollable.tsx

```diff
@@ -188,13 +270,24 @@ export class Scrollable extends React.PureComponent<Props> {
     return (
       <ScrollableUI
         {...getValidProps(rest)}
+        fadeSides={fadeLeft || fadeRight}
         className={componentClassName}
         innerRef={innerRef}
       >
+        {(fadeLeft || fadeRight) && (
+          <EventListener
+            event="resize"
+            handler={() => {
```

Comment:
@tinkertrain Would you be able to make this a class method 😊 (for performance reasons). Creating an inline-arrow functions means a new `Object` every time, which triggers prop change, which triggers React render cycle stuff 🤣

Reply:
Good catch, just changed!